docs/src/pages/[platform]/theming/theming.react.mdx

```diff
@@ -78,9 +78,7 @@ export const App = () => {
 
 ### Design Tokens
 
-Amplify UI uses [Design Tokens](https://www.designtokens.org/) for storing design decisions and is the primary way to theme the components.
-
-Amplify UI follows the [System UI Theme Specification](https://system-ui.com/theme/) with some modifications. The System UI Theme Specification outlines top-level namespaces for categorizing design token types. For example, colors go under the `colors` namespace. This specification is used by [Stitches](https://stitches.dev/docs/tokens), [Chakra-UI](https://chakra-ui.com/docs/styled-system/theme), and [Evergreen](https://evergreen.segment.com/introduction/theming).
+Amplify UI uses [Design Tokens](https://www.designtokens.org/) for storing design decisions and is the primary way to theme the components. Design tokens are categorized by type under namespaces; for example, colors go under the `colors` namespace. This specification is used by [Stitches](https://stitches.dev/docs/tokens), [Chakra-UI](https://chakra-ui.com/docs/styled-system/theme), and [Evergreen](https://evergreen.segment.com/introduction/theming).
```

Comment:
Think this reads a little bit differently now; it's now saying Stitches, Chakra, and Evergreen use Design Tokens instead of System UI design specification. That is not necessarily true, right?

Reply:
I updated this in the latest to remove reference to "specification". TBH I'm not sure how closely any of us follow the proposed specification now; but we do organize our design tokens in a similar way so I re-worded that part.